python-sdk/langwatch/src/langwatch/utils/transformation.py

```
@@ -0,0 +1,218 @@
+                TypeAdapter(type_).validate_json(
+                    json.dumps(item, cls=SerializableWithStringFallback)
+                )
+                return True
+            except (ValidationError, AttributeError, TypeError):
+                return False
+
+    return False
+
+
+def rag_contexts(value: Union[List[RAGChunk], List[str]]) -> List[RAGChunk]:
+    if type(value) == list and all(
```

Ruff (0.8.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)